src/azul/service/responseobjects/elastic_request_builder.py

```diff
@@ -584,7 +590,11 @@ def transform_manifest(
             filters = {"file": {}}
         # Create an ElasticSearch request
         filters = filters['file']
-        es_search = self.create_request(filters, self.es_client, request_config, post_filter=False)
+        es_search = self.create_request(filters,
+                                        self.es_client,
+                                        request_config,
+                                        post_filter=False,
+                                        source_filter=['bundles.*', 'contents.files.*'])
```

Comment:
The manifest fields are defined in `request_config.json`. Clearly there is coupling between this source filter and that configuration. Can you see if you can derive the source filter from the configuration? If that's too involved file a ticket and comment on the coupling here, referencing the ticket.